src/main/resources/biz_fr.properties

```
@@ -54,7 +54,7 @@ AverageActivityForTenantChart.label = Activité moyenne
 AverageEducationForTenantChart.chartDescription = Affiche le niveau moyen de formation des utilisateurs sous forme de pourcentage. À cette fin, le pourcentage de vidéos disponibles déjà visionnées par utilisateur est pris en compte.
 AverageEducationForTenantChart.description = Affiche le niveau moyen de formation des utilisateurs sous forme de pourcentage.
 AverageEducationForTenantChart.label = Niveau de formation moyen
-BaseFileParameter.invalidFileExtension = Le fichier "${name}" ne contient aucune des extensions de fichier valides : ${extensions}.
+BaseFileParameter.invalidFileExtension = Le fichier '${nom}' n'a aucune des extensions de fichier valides : ${extensions}
```

```suggestion
BaseFileParameter.invalidFileExtension = Le fichier '${name}' n'a aucune des extensions de fichier valides : ${extensions}
```
also fixed in lokalise